### ISSUE TITLE
How to add `rustup` targets?

### ISSUE THREAD

**Issue by JohnEmhoff** [panamax-rs/panamax#52]
Hello! How do I configure panamax to pull down additional compilation targets? That is, things that you would normally get with something along the lines of `rustup target add aarch64-linux-android`. I figured `pinned_rust_versions` was the closest option, but if it is I can't seem to find the right settings (`stable-aarch64-linux-android` doesn't seem to work, for instance).

**Comment by k3d3** [comment]
Hello! I believe you would need to add that to `platforms_unix`.

`pinned_rust_versions` is more for e.g. pinning Rust 1.46 to make sure it doesn't get deleted when a newer stable version is released.

**Comment by JohnEmhoff** [comment]
That does it, thanks!

**Comment by FauxPrada** [comment]
Is it possible for this to work with ARM targets? putting them in "platforms_unix" or windows doesnt work. Neither did making a new "platforms_arm" (just in case :P)

**Comment by k3d3** [comment]
@FauxPrada Apologies for the late response - this is because we have a hardcoded list of available platforms [here](https://github.com/panamax-rs/panamax/blob/master/src/rustup.rs#L23) and [here](https://github.com/panamax-rs/panamax/blob/master/src/rustup.rs#L102). I believe `thumbv7em-none-eabihf` doesn't exist there either because I've missed it, or because it got added since the last time I modified that list.

That being said, having a hardcoded list restrict which platforms can be downloaded doesn't make a whole lot of sense, so I'm going to make sure other platforms can be added outside of that list.
